Translator/Translator/Source File/reader.cpp

```diff
@@ -46,3 +46,8 @@ vector<Point> Reader::read() {
 vector<double> Reader::get_unique_points() {

```

Comment:
Point should contain index of unique double value instead of unique double value itself.

Reply:
in point.cpp
I done the changes

Reply:
changes done in point.cpp